[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/big_query_setup.md

    * Customer managed key
* Create a table ‘events’
    * Export the schema from an existing table.

>         * Install the gcloud CLI if you don’t have it (https://cloud.google.com/sdk/docs/install)

Reviewer: ```suggestion
        * Install the gcloud CLI if you do not have it (https://cloud.google.com/sdk/docs/install)
```
